### ISSUE TITLE
Wrong import stripe_price

### ISSUE THREAD

**Issue by galkin** [lukasaron/terraform-provider-stripe#42]
@lukasaron thank your work! There is one more bug report.

Reproducing steps:
1. Create product and price
```terraform
resource "stripe_product" "product" {
  name = "minimalist product"
}

resource "stripe_price" "price" {
  product        = stripe_product.product.id
  currency       = "usd"
  active         = true
  billing_scheme = "per_unit"
  unit_amount    =  14999

  recurring {
    interval       = "month"
    interval_count = 1
    usage_type     = "licensed"
  }
}
```
2. Run `terraform console` and check stripe_price.price resource. unit_amount will be correct. Record id for future import.
3. Run `terraform state rm stripe_price.price`
4. Run `terraform import stripe_price.price price_XXX`. Use id from step 2.
5. Run `terraform console` and check stripe_price.price resource.  unit_amount will be `tonumber(null)` instead of correct value.

**Comment by galkin** [comment]
@lukasaron, do you have any updates?

**Comment by lukasaron** [comment]
@galkin Thanks for the steps! This helped a lot with the investigation.

The fix should be in version v1.7.6.

**Comment by galkin** [comment]
@lukasaron confirmed. the issue is fixed. Thank you!
